src/components/navbar/navbar.scss

```diff
@@ -0,0 +1,149 @@
+@import '../../assets/base.scss';
+
+nav {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border-right: $dark-grey 1px solid;
+  z-index: 100;
+  min-height: 100vh;
+  max-width: 20vw;
+}
+
+footer {
+  margin: 2rem 0;
+  row-gap: 5px;
+}
+
+small {
+  font-size: small;
+  color: $dark-grey;
+}
+
+nav img {
+  margin: 10px 0;
+  width: 100%;
+}
+
+nav.green {
+  background-color: #fff;
+}
+
+.side-bar {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin-left: 1rem;
+}
+
+.link {
+  color: $text-grey;
+}
+
+.side-bar .link {
+  border-radius: 0;
+  margin-bottom: 0.25rem;
+}
```

Comment:
[OPTIONAL] Please make sure to use the same font/font-weight as shown on the design:
![image](https://user-images.githubusercontent.com/42799579/197821909-d00ff6fe-cb25-4c1c-90f9-1b5323a71805.png)

![image](https://user-images.githubusercontent.com/42799579/197821803-5001d868-5e81-4009-9aac-2142a6e4a991.png)

Otherwise the project looks amazing 😉.

Reply:
![image](https://user-images.githubusercontent.com/59999191/197844520-a3b523e8-ab13-455b-bbc9-8d98dcdaf549.png)
 
 Thanks for the suggestion